src/features/pages/ctrader-signup/index.tsx

```diff
@@ -8,11 +8,14 @@ import Link from 'features/components/atoms/link'
 import useRegion from 'components/hooks/use-region'
 import CtraderWrapper from 'features/components/templates/ctrader/ctrader-wrapper'
 import Layout from 'features/components/templates/layout'
+import { ctrader_login, brand_name } from 'common/constants'
+import { getLanguage } from 'common/utility'
 
 const CtraderSignup = () => {
     const { is_eu } = useRegion()
 
     const security_pdf_link = `/tnc${is_eu ? '/eu' : ''}/security-and-privacy.pdf`
+    getLanguage()
```

Comment:
Is there a reason to call this here?

Reply:
Looks like, it's getting used in the url
href: `${ctrader_login}&brand=${brand_name.toLowerCase()}&source=web&lang=${getLanguage()}`,
                                    }}